src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,37 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+
 public class FileWork {
+    private static final String REGEX_SPLITTER = "\\W+";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        String[] result = {};
+        StringBuilder stringBuilder = new StringBuilder();
```

Comment:
```suggestion
        StringBuilder filteredWords = new StringBuilder();
```
let's improve naming